campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -217,7 +189,7 @@
       <button class="button donation-amount donation-amount-third" data-amount="{{ donationAmountThird }}"></button>
       <button class="button donation-amount donation-amount-fourth" data-amount="{{ donationAmountFourth }}"></button>
 
-      <a class="button button-red button-wide donate-button  {% if buttonAnimate %} {{ buttonAnimate }} {% endif %}" href="{{ donationFormURL|safe }}">
```

Comment:
I see that you remove `buttonAnimate` variable. I'm verifying that that's OK since this is not used by any of the snippets based on this template.